src/processes/follow-topic-process.js

```diff
@@ -146,18 +149,21 @@ const followTopicProcess = async (job, done) => {
           } catch (error) {
             console.log('::: Error on stopWatching :::', error);
           }
-        } else if (
-          channelState.messages.length === 1 &&
-          channelState.messages[0]?.is_auto_message
-        ) {
-          // add new condition based on ticket ping-4227
-          // check if first message is auto message from sender
+          // } else if (channelState.messages.length <= 1 && channelState.messages[0]?.is_auto_message) {
+          //   // add new condition based on ticket ping-4227
+          //   // check if first message is auto message from sender
+          //   await chat.sendMessage({
+          //     text: communityMessageFormat.message,
+          //     user_id: communityMessageFormat.user_id,
+          //     is_auto_message: true
+          //   });
+        } else {
+          // console.log(':::  Channel not eligible to receive auto message :::');
           await chat.sendMessage({
             text: communityMessageFormat.message,
-            user_id: communityMessageFormat.user_id
+            user_id: communityMessageFormat.user_id,
+            is_auto_message: true
           });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented-out code should be removed. It's generally not a good practice to leave commented-out code in the codebase, as it can confuse other developers and make the code harder to read. If this code is needed for future reference, it should be documented elsewhere (e.g., in a Git commit message or documentation).
```diff
- // } else if (channelState.messages.length <= 1 && channelState.messages[0]?.is_auto_message) {
- //   // add new condition based on ticket ping-4227
- //   // check if first message is auto message from sender
- //   await chat.sendMessage({
- //     text: communityMessageFormat.message,
- //     user_id: communityMessageFormat.user_id,
- //     is_auto_message: true
- //   });
- } else {
-   // console.log(':::  Channel not eligible to receive auto message :::');
-    await chat.sendMessage({
-      text: communityMessageFormat.message,
-     user_id: communityMessageFormat.user_id,
-     is_auto_message: true
-    });
+ await chat.sendMessage({
+   text: communityMessageFormat.message,
+   user_id: communityMessageFormat.user_id,
+   is_auto_message: true
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->